src/apps/HALOEXCHANGE_FUSED-OMP.cpp

```diff
@@ -91,7 +92,8 @@ void HALOEXCHANGE_FUSED::runOpenMPVariant(VariantID vid, size_t RAJAPERF_UNUSED_
             buffer += len;
           }
         }
-#if _OPENMP >= 200805
+
+#if defined(RAJA_ENABLE_OMP_TASK_INTERNAL) 
```

Comment:
Nice!

Reply:
There is an exposed CMake option for this 'RAJA_ENABLE_OMP_TASK' in RAJA and a corresponding Spack variant in the RAJA and RAJAPerf spack packages. The result is that we can enable/disable internal RAJA algorithm variants for things like OpenMP sort for different builds/tests based on compiler.  We didn't see the need for this before, but something is fishy with newer clang compilers on blueos systems.